docs/topics/con_server-sent-events.adoc

```diff
@@ -4,7 +4,7 @@
 Server-sent events (SSE) is a push technology allowing HTTP sever to send unidirectional updates to the client.
 SSE works by establishing a connection between the event source and the client.
 The event source uses this connection to push events to the client-side.
-After the server pushes the events, the connection remains open, and can be used used to push subsequent events.
+After the server pushes the events, the connection remains open, and can be used to push subsequent events.
```

Comment:
```suggestion
After the server pushes the events, the connection remains open and can be used to push subsequent events.
```

Reply:
Good catch @huntley :) Thank you, accepted.